### DISCUSSION TITLE
start container failed, iptable : network not found

### DISCUSSION THREAD

**Discussion by cache-missing** [wg-easy/wg-easy#994] [Q&A]
**Describe the bug**
iptables v1.8.10 (legacy): host/network `fcca::1' not found
Try `iptables -h' or 'iptables --help' for more information.
[#] ip link delete dev wg0

    at ChildProcess.exithandler (node:child_process:422:12)
    at ChildProcess.emit (node:events:517:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:303:5) {
  code: 2,
  killed: false,
  signal: null,
  cmd: 'wg-quick up wg0'
}
and from log, it seems that the WG_PORT is also not configured

**To Reproduce**
use cmd 
```shell
  docker run \
  --name=wg-easy \
  -e LANG=chs \
  -e WG_HOST=xx.xx.xx.xx \
  -e WG_PORT=1888 \
  -e PASSWORD=xxxx \
  -e WG_DEFAULT_ADDRESS="fcca::1" \
  -v $HOME/services-inuse/wireguard:/etc/wireguard \
  -p 1888:1888/udp \
  -p 51821:51821/tcp \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --sysctl="net.ipv6.conf.all.forwarding=1" \
  --sysctl="net.ipv6.conf.all.disable_ipv6=0" \
  --rm \
  ghcr.io/wg-easy/wg-easy
```

**Screenshots**
<img width="1694" alt="image" src="https://github.com/wg-easy/wg-easy/assets/90820067/41d9fbda-2d6b-4d3c-b628-c2642d8f52a0">

**Desktop (please complete the following information):**
 - OS: linux

**Comment by pheiduck** [comment]
Until now we don't support IPv6. #781 
Please use IPv4.

**Comment by sshpem** [comment]
you use iptables, but fcca::1 is ipv6 address.
